assets/js/app.ts

```diff
@@ -229,6 +231,16 @@ const Hooks = {
         this.player.streamType = opts.is_live ? "ll-live:dvr" : "on-demand";
         this.player.src = opts.url;
 
+        this.player.addEventListener("provider-change", (event) => {
+          const provider = event.detail;
```

Comment:
Sorry for commenting on a closed PR, but trying to understand how this works.

Since `provider` goes out of scope in line 241, does setting the target latency have any effect?

Maybe I'm missing something, but AFAICT, this event listener doesn't do anything.

Reply:
Ah, I think I understand: the listener is being passed a reference to the `MediaProviderAdapter` which this function mutates.